app/views/_cascade/formats/modular/_page_wrapper_open.erb

```diff
@@ -0,0 +1,19 @@
+<%
+
+# VTL variables
+baseClass = ''
+sectionClasses = ''
+currentPagePath = @current_page_path || '/'
+
+currentPagePath.split('/').each do |name|
+  if currentPagePath.split('/').first == name
+    baseClass = name
+    sectionClasses = baseClass
+  elsif currentPagePath.split('/').last != name
+    baseClass = "#{baseClass}__#{name}"
+    sectionClasses = "#{sectionClasses} #{baseClass}"
+  end
+end
```

Comment:
This tries to replicate some VTL code in Cascade. So I think it follow their logic. In any event, it works for our purposes here -- testing themes -- and doesn't get folded back into anything functional. I treat it more as a test helper.

I'll add a comment to clarify that.